### PR TITLE
Panther ros

### PR DESCRIPTION
**Pull request by KmakD**
Initial draft of repo for all panther packages

### CODE REVIEW

File: panther_bringup/config/panther_default.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  imu:
+    model: "PhidgetSpatial_1042_0B"
+    localization:
+      x: 0.195
+      y: -0.01
+      z: 0.085
+    orientation:
+      r: 3.14159
+      p: 0.0
+      y: 3.14159
```
Comment (Kotochleb) [inline panther_bringup/config/panther_default.yaml:15]: This requires double check

**Review by Kotochleb — CHANGES_REQUESTED** [review]

File: panther_bringup/config/panther_default.yaml (hunk 1 above)
Comment (Kotochleb) [reply panther_bringup/config/panther_default.yaml:15]: This config is for Panther 1.0.5
Parameters for 1.0 can be found here:
https://github.com/husarion/panther_driver/blob/1040796540c970d60c88310f96c7181c263ba747/launch/phidgets_imu.launch#L20

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+.vscode
+__pycache__/
```
[the remark this answers is not in the document]
Comment (KmakD) [reply .gitignore:2]: :+1:

File: panther/package.xml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  <url type="repository">https://github.com/husarion/panther_ros</url>
+  <url type="bugtracker">https://github.com/husarion/panther_ros/issues</url>
+
+  <buildtool_depend>catkin</buildtool_depend>
+
+  <depend>panther_bringup</depend>
+  <depend>panther_driver</depend>
+  <depend>panther_description</depend>
+  <depend>panther_lights</depend>
+  <depend>panther_msgs</depend>
```
[the remark this answers is not in the document]
Comment (KmakD) [reply panther/package.xml:22]: :

File: panther_bringup/package.xml [hunk 4]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>panther_bringup</name>
+  <version>0.0.0</version>
+
+  <description>The panther_bringup package</description>
```
[the remark this answers is not in the document]
Comment (KmakD) [reply panther_bringup/package.xml:6]: :+1:

File: panther_bringup/config/panther_default.yaml (hunk 1 above)
Comment (KmakD) [reply panther_bringup/config/panther_default.yaml:15]: The values you send seems to be good.

**Pull request merged**